tools/autograd/gen_variable_type.py

```diff
@@ -510,6 +518,48 @@ def format_trace(declaration):
     return (format_prerecord_trace(declaration), format_postrecord_trace(declaration))
 
 
+# Methods shared by TraceType and VariableType to handle return variable declaration, tie and tuple.
+def format_return_variables(declaration):
```

Comment:
It looks like this was refactored from https://github.com/pytorch/pytorch/pull/38467/files#diff-bbfdc6df0a634ed4c7442fdc4129499dL1003 ; was there any substantive change in this refactor?

Reply:
No. BTW, you can review this on D21570684 - in Phabricator moved lines are highlighted with orange bar after line number :)